CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+
+If no existing item describes your issue/feature, great - please file a new issue:
+
+### File a new Issue
+
+* Don't know whether you're reporting an issue or requesting a feature? File an issue
+* Have a question that you don't see answered in docs, videos, etc.? File an issue
+* Want to know if we're planning on building a particular feature? File an issue
+* Got a great idea for a new feature? File an issue/request/idea
+* Don't understand how to do something? File an issue
+* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+
+When you hit "New Issue", select the type of issue closest to what you want to report/ask/request to be prompted with the appropriate issue template.
+
+### Complete the template
+
+**Complete the information requested in the issue template, providing as much information as possible**. The more information you provide, the more likely your issue/ask will be understood and implemented. 
+
+* **We LOVE detailed repro steps!** What steps do we need to take to reproduce the issue? Assume we love to read repro steps. As much detail as you can stand is probably _barely_ enough detail for us!
+* Prefer error message text where possible or screenshots of errors if text cannot be captured
+* We MUCH prefer text command-line script than screenshots of command-line script.
+* **If you intend to implement the fix/feature yourself then say so!** If you do not indicate otherwise we will assume that the issue is our to solve, or may label the issue as `Help-Wanted`.
+
+---
+
+## Contributing fixes / features
+
+If you're able & willing to help fix issues and/or implement features, we'd love your contribution!
+
+The best place to start is the list of ["good first issue"](https://github.com/microsoft/react-native-windows/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) issues. These are bugs or tasks that we on the team believe could be implemented for someone without any prior experience in the codebase. Once you're feeling more comfortable in the codebase, feel free to just use the ["Help Wanted"](https://github.com/microsoft/react-native-windows/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+) label, or just find an issue your interested in and hop in!
+
+Often, we like to assign issues that generally belong to somebody's area of expertise to the team member that owns that area. This doesn't mean the community can't jump in -- they should reach out and have a chat with the assignee to see if it'd okay to take. If an issue's been assigned more than a month ago, there's a good chance it's fair game to try yourself.
+
+For a primer on our codebase, see ["Repository Contents"](https://github.com/microsoft/react-native-windows/wiki/Repository-Contents) and ["Project Structure."](https://github.com/microsoft/react-native-windows/blob/main/docs/project-structure.md)
+
+---
+
+## 👨‍💻Development
```

Comment:
nit, some headers have a space after the emoji and some don't, make them consistent :)

Reply:
Thanks! Went with spacing :)